frb_example/pure_dart_multi/rust/src/fake_module/mod.rs

```diff
@@ -0,0 +1,7 @@
+/// A fake module, for testing `import` statement(even it is not used) in regular API .rs file.
+/// Currently, if there is no valid content in a rust file,
+/// frb would panic. This panic is expected in release mode, but not
+/// in debug mode, so here some dummy content is added.
```

Comment:
nit: from the comments we are testing "import" - so I guess in reality, we are going to import some real things? so shall we add something here and *really use* it somewhere to maximally mimic the real scenario